src/Raven.Server/Documents/ConflictsStorage.cs

```diff
@@ -706,30 +667,42 @@ public void ThrowConcurrencyExceptionOnConflict(long? expectedEtag, long? curren
                     collection = incomingTombstoneCollection;
                 }
 
-                using (Slice.From(context.Allocator, incomingChangeVector, out var cv))
-                using (DocumentIdWorker.GetStringPreserveCase(context, collection, out Slice collectionSlice))
-                using (conflictsTable.Allocate(out TableValueBuilder tvb))
-                {
-                    tvb.Add(lowerId);
-                    tvb.Add(SpecialChars.RecordSeparator);
-                    tvb.Add(cv);
-                    tvb.Add(idPtr);
-                    tvb.Add(doc, docSize);
-                    tvb.Add(Bits.SwapBytes(etag));
-                    tvb.Add(collectionSlice);
-                    tvb.Add(lastModifiedTicks);
-                    tvb.Add((int)flags);
-                    if (conflictsTable.Set(tvb))
-                        Interlocked.Increment(ref ConflictsCount);
-                }
+                AddToConflictsTable(incomingChangeVector,
```

Comment:
Actually it would be great if we can unify all other tables as well like that

Reply:
added a comment in the issue page to address this when working on bucket-stats